keras_app/views/layers_export.py

```diff
@@ -642,6 +643,16 @@ def time_distributed(layerId, idNext, net, layer_in, layer_map):
     out = {}
     out[layerId] = TimeDistributed(
         layer_map[net[idNext]['info']['type']](net[idNext], layer_in, idNext, False)[idNext])(*layer_in)
+
+
+# LRN for Tensorflow export only
```

Comment:
Seeing that we use the Keras backend to export TF, and the custom layer is written in Keras, I feel there should be a way to use it for Keras export as well. Can you check this once?

Reply:
In the latest commit it's possible to use it with Keras too. I just forgot to delete that comment.